src/components/markdown-htmlview.component.js

```diff
@@ -0,0 +1,311 @@
+import React, { Component } from 'react';
+import { Dimensions, StyleSheet, View, Text, Platform } from 'react-native';
+import HTMLView from 'react-native-htmlview';
+import marked from 'marked';
+
+import { ImageZoom } from 'components';
+import { colors, emojis, fonts, normalize } from 'config';
+
+const lightFont = {
+  ...fonts.fontPrimaryLight,
+};
+
+const regularFont = {
+  ...fonts.fontPrimary,
+};
+
+const textStyleLight = {
+  fontSize: Platform.OS === 'ios' ? normalize(11) : normalize(12),
+  color: colors.primaryDark,
+  ...lightFont,
+};
+
+const textStyleRegular = {
+  fontSize: Platform.OS === 'ios' ? normalize(11) : normalize(12),
+  color: colors.primaryDark,
+  ...regularFont,
+};
+
+const textStyle = Platform.OS === 'ios' ? textStyleLight : textStyleRegular;
+
+const linkStyle = {
+  color: colors.primaryDark,
+  ...fonts.fontPrimarySemiBold,
+};
+
+const styles = StyleSheet.create({
+  span: textStyle,
+  p: {
+    padding: 0,
+    margin: 0,
+    ...textStyle,
+  },
+  strong: {
+    ...textStyle,
+    ...fonts.fontPrimaryBold,
+  },
+  em: {
+    ...textStyle,
+    ...fonts.fontPrimaryBold, // FIXME: we need an italic font
```

Comment:
Should italic font go here?

Reply:
yes, to render *this markdown* correctly. We need to bundle it like we do for other variants